app/frontend/src/pages/Group/Join/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+
+import { Button, Input } from '@morak/ui';
+
+import { GroupInfo } from '@/components/GroupInfo';
+import { fontStyle } from '@/styles';
+
+import * as styles from './index.css';
+
+export function GroupJoinPage() {
+  const { sansBold36, sansBold24, sansRegular18 } = fontStyle;
+
+  const [currentStep, setCurrentStep] = useState(1);
+
+  const getCurrentState = (step: number) => {
+    if (step > currentStep) {
+      return 'disabled';
+    }
+
+    if (step < currentStep) {
+      return 'completed';
+    }
+
+    return '';
+  };
+
+  return (
+    <div className={styles.container}>
+      <h2 className={sansBold36}>그룹 참여</h2>
+      <section className={`${styles.section} ${getCurrentState(1)}`}>
+        <h3 className={sansBold24}>1. 참여 방법 선택</h3>
+        <p className={sansRegular18}>어떤 방법으로 그룹에 참여할까요?</p>
+        <div className={styles.confirmButtons}>
+          <Button
+            theme="primary"
+            shape="line"
+            size="large"
+            fullWidth
+            onClick={() => setCurrentStep(2)}
+          >
+            참여 코드로 참여할래요
+          </Button>
+          <NavLink to="/groups" className={styles.navLinkButton}>
+            <Button
+              type="button"
+              theme="primary"
+              shape="line"
+              size="large"
+              fullWidth
+            >
+              공개된 그룹을 둘러볼래요
+            </Button>
+          </NavLink>
+        </div>
+      </section>
```

Comment:
css로 컴포넌트의 렌더링을 조정하는 것보다 js로 이렇게 쓰는 게 나을 것 같아요. 
혹시 다음과 같은 코드를 작성한 이유가 있다면 알려 주세요!
```suggestion
      {currentStep === 1 && (
      <section className={styles.section}>
        <h3 className={sansBold24}>1. 참여 방법 선택</h3>
        <p className={sansRegular18}>어떤 방법으로 그룹에 참여할까요?</p>
        <div className={styles.confirmButtons}>
          <Button
            theme="primary"
            shape="line"
            size="large"
            fullWidth
            onClick={() => setCurrentStep(2)}
          >
            참여 코드로 참여할래요
          </Button>
          <NavLink to="/groups" className={styles.navLinkButton}>
            <Button
              type="button"
              theme="primary"
              shape="line"
              size="large"
              fullWidth
            >
              공개된 그룹을 둘러볼래요
            </Button>
          </NavLink>
        </div>
      </section>
      )}
```

Reply:
아 이거 미처 놓치고 지나간 부분인데 지나간 부분은 opacity가 낮아지면서 노출이 되는 거였군요... 
두 방법 다 아이디어는 좋은 것 같아요 👍